src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -328,7 +331,8 @@ function MoneyRequestPreview(props) {
                                     {!isCurrentUserManager && props.shouldShowPendingConversionMessage && (
                                         <Text style={[styles.textLabel, styles.colorMuted]}>{translate('iou.pendingConversionMessage')}</Text>
                                     )}
-                                    {(shouldShowDescription || shouldShowMerchant) && <Text style={[styles.textLabelSupporting, styles.textNormal]}>{merchantOrDescription}</Text>}
+                                    {shouldShowDescription && <RenderHTML html={parser.replace(merchantOrDescription)} />}
```

Comment:
Changing text to html caused regression of link not wrapping - https://github.com/Expensify/App/issues/34952

![298867450-9cce09e8-b810-4561-87a4-08183f10a29d](https://github.com/Expensify/App/assets/108292595/957f62b8-33b0-4e02-8a0d-28ccc61fe238)